lang/kotlin/kotlin.py

```diff
@@ -0,0 +1,182 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+ctx.matches = r"""
+tag: user.kotlin
+"""
+
+
+@ctx.action_class("user")
+class UserActions:
+    def code_comment_line_prefix():
+        actions.insert("// ")
+
+    def code_operator_indirection():
+        actions.skip()
+
+    def code_operator_address_of():
+        actions.skip()
+
```

Comment:
Actions shouldn't be implemented if they're not implementable. This suppresses the warning about the action being unimplemented when used. We don't `actions.skip()` anywhere else in `lang/`.

```suggestion
```

Reply:
Removed.